zerver/lib/import_realm.py

```diff
@@ -681,7 +693,7 @@ def import_uploads(import_dir: Path, processing_avatars: bool=False,
 # Because the Python object => JSON conversion process is not fully
 # faithful, we have to use a set of fixers (e.g. on DateTime objects
 # and Foreign Keys) to do the import correctly.
-def do_import_realm(import_dir: Path, subdomain: str) -> Realm:
+def do_import_realm(import_dir: Path, subdomain: str, threads: int) -> Realm:
```

Comment:
Let's spell this as `threads: int=1` and the not change all the tests to pass this argument.